controllers/controllers/workloads/env/vcap_app_builder_test.go

```diff
@@ -63,5 +81,70 @@ var _ = Describe("VCAP_APPLICATION env value builder", func() {
 				Expect(vcapAppValue).To(HaveKeyWithValue("x", HaveKeyWithValue("y", "z")))
 			})
 		})
+
+		When("application routes are provided", func() {
+			var (
+				appDomain *korifiv1alpha1.CFDomain
+				appRoute  *korifiv1alpha1.CFRoute
+				routeUri  string
+			)
+
+			BeforeEach(func() {
+				appDomain = &korifiv1alpha1.CFDomain{
```

Comment:
Korifi does mandate a domain to exist when creating a route. However, in this particular test suite, the route validator is not installed, therefore I believe that the route's `DomainRef` could be left empty. Then, creating the domain is no longer needed - this would make the test shorter and more focused.